### PR TITLE
standard deployment analytics pings

### PR DESCRIPTION
**Pull request by elharo**
@chanseokoh

### CODE REVIEW

**Comment by chanseokoh** [comment]
I like going with "standard" as a metadata key (i.e., using it as a label). I'll create an issue to refactor existing pings, which currently does "type=maven" or "type=native".

I'll do some tests later today.

File: plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java [hunk 1]
```diff
@@ -96,16 +98,19 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 
   private void launchDeployJob(IProject project, Credential credential, ExecutionEvent event) 
       throws IOException, ExecutionException {
+    
+    AnalyticsPingManager.getInstance().sendPing(AnalyticsEvents.APP_ENGINE_DEPLOY, "standard", null);
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java]: Why not adding "standard" as a named constant in `AnalyticsEvent`?

Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java [hunk 2]
```diff
@@ -114,6 +120,8 @@ private void launchDeployJob(IProject project, Credential credential, ExecutionE
       @Override
       public void done(IJobChangeEvent event) {
         super.done(event);
+        AnalyticsPingManager.getInstance()
+            .sendPing(AnalyticsEvents.APP_ENGINE_DEPLOY_SUCCESS, "standard", null);
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java]: Should have mentioned here too.

Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java]: done

**Pull request merged**